R/vertical.R

```diff
@@ -5,7 +5,7 @@
 #' @param x A character vector or an object of class "vertical".
 #' @keywords internal
 construct_vertical <- function(x) {
-  stopifnot(inherits(x, what = c("utf8", "character", "vertical")))
+  stopifnot(is.character(x) || inherits(x, what = c("utf8", "vertical")))
```

Comment:
```suggestion
  stopifnot(is.character(x))
```
Can we further simplify like this? Because 
```r
all(
  is.character(glue::glue("1")),
  is.character(styler:::construct_vertical("1")),
  is.character(enc::as_utf8('3'))
)
#> TRUE
```

Reply:
Works for me